core-feature-pack/src/main/resources/content/bin/jboss-cli.xml

```diff
@@ -47,12 +47,23 @@
 
     <!-- Include the prompt with the command into the output for each command executed in non-interactive mode -->
     <echo-command>false</echo-command>
-    
+
     <!-- Uncomment to set the command timeout. Element value is in seconds -->
     <!-- <command-timeout>30</command-timeout> -->
 
     <!-- Uncomment to display operation responses using JSON syntax.
     By default responses are displayed using DMR string syntax. -->
     <!-- <output-json>true</output-json> -->
-    
+
+    <!-- Uncomment to display CLI output in colour and configure each of the available responses.
+    Available colors: black, red, green, yellow, blue, magenta, cyan, white and default, which is the terminal's default foreground color-->
+    <!--<color-output>-->
```

Comment:
Perhaps a single comment would simplify uncommenting this part

Reply:
Like so:
```xml
    <!-- Uncomment to display CLI output in colour and configure each of the available responses. -->
    <!-- Available colors: black, red, green, yellow, blue, magenta, cyan, white and default, which is the terminal's default foreground color-->
```